src/pages/workspace/companyCards/WorkspaceCompanyCardsListHeaderButtons.tsx

```diff
@@ -1,48 +1,47 @@
 import React from 'react';
 import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
 import Button from '@components/Button';
 import CaretWrapper from '@components/CaretWrapper';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import {PressableWithFeedback} from '@components/Pressable';
 import Text from '@components/Text';
 import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
 import useResponsiveLayout from '@hooks/useResponsiveLayout';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as CardUtils from '@libs/CardUtils';
 import Navigation from '@navigation/Navigation';
 import variables from '@styles/variables';
+import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
-import type {CardFeeds} from '@src/types/onyx';
-
-const mockedFeeds = {
-    companyCardNicknames: {
-        cdfbmo: 'BMO MasterCard',
-    },
-} as unknown as CardFeeds;
 
 type WorkspaceCompanyCardsListHeaderButtonsProps = {
     /** Current policy id */
     policyID: string;
 
     /** Currently selected feed */
     selectedFeed: string;
+
+    /** Is pending state of the feed */
```

Comment:
Can you reword this please? Also what does it mean for the feed to be pending?

Reply:
Pending for feed means that user added a feed but its under the review

Reply:
ok then I suggest we change the comment to: `Is the feed pending review?` or something similar